src/components/Register.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { motion } from "framer-motion";
+import classes from "./Login.module.css";
+import logo from "../images/logo.png";
+
+const Register = () => {
+  return (
+    <motion.div
+      className={classes.login_section}
+      initial={{ width: 0, opacity: 0 }}
+      animate={{
+        width: "auto",
+        opacity: 1,
+        textAlign: "center",
+        transition: { duration: 0.5 },
+      }}
+      exit={{ x: window.innerWidth, opacity: 0, transition: { duration: 0.5 } }}
+    >
+      <div className={classes.overlay}>
+        <div className={classes.logo}>
+          <img src={logo} />
+        </div>
+        <div className={classes.login_form}>
+          <h1>
+            Create new Account <span className={classes.dot}>.</span>
+          </h1>
```

Comment:
You should wrap this whole list of inputs into`<form>` and you could use `Formik` library for handling form data sumbitting.

Relevant link: https://formik.org/docs/overview